src/errors.js

```diff
@@ -82,9 +82,15 @@ export function onlyErrorsWithCode(
   return (error) => {
     let throwError = true;
 
-    if (Array.isArray(codeWanted) && codeWanted.indexOf(error.code) !== -1) {
-      throwError = false;
-    } else if (error.code === codeWanted) {
+    if (Array.isArray(codeWanted)) {
+      if (codeWanted.indexOf(error.code) !== -1) {
+        throwError = false;
+      }
+      if (codeWanted.indexOf(error.errno) !== -1) {
```

Comment:
You can group these statements to make the code more concise and less repetitive. Example:
````javascript
if (codeWanted.indexOf(error.code) !== -1 ||
    codeWanted.indexOf(error.errno) !== -1) {
  throwError = false;
}